### PR TITLE
Fix UI with updated JSON for VOI

### PR DESCRIPTION
**Pull request by sudhirverma**

### CODE REVIEW

File: src/data-store/dataStore.ts [hunk 1]
```diff
@@ -5,4 +5,4 @@ export const version_details =
 export const assessment_datastore =
   "https://raw.githubusercontent.com/Be-Secure/besecure-assessment-datastore/main";
 export const vulnerabilityOffInterest =
-  "https://raw.githubusercontent.com/sudhirverma/besecure-ossvoi-datastore/main/vulnerability-of-interest.json";
+  "https://raw.githubusercontent.com/sudhirverma/besecure-ossvoi-datastore/adding_new_data/vulnerability-of-interest.json";
```
Comment (asa1997) [inline src/data-store/dataStore.ts:8]: Please change the namespace to Be-Secure

**Review by asa1997 — CHANGES_REQUESTED** [review]
@sudhirverma Please address the comment.

File: src/data-store/dataStore.ts (hunk 1 above)
Comment (sudhirverma) [reply src/data-store/dataStore.ts]: done

**Review by asa1997 — APPROVED** [review]
@sudhirverma LGTM

**Pull request merged**